eth/gasprice/feehistory.go

```diff
@@ -316,13 +334,13 @@ func (oracle *Oracle) FeeHistory(ctx context.Context, blocks uint64, unresolvedL
 		}
 	}
 	if firstMissing == 0 {
-		return common.Big0, nil, nil, nil, nil
+		return common.Big0, nil, nil, nil, nil, nil, nil
 	}
 	if len(rewardPercentiles) != 0 {
 		reward = reward[:firstMissing]
 	} else {
 		reward = nil
 	}
 	baseFee, gasUsedRatio = baseFee[:firstMissing+1], gasUsedRatio[:firstMissing]
```

Comment:
I think I need to do this `[:firstMissing]` slicing on `blobBaseFee` and `blobGasUsedRatio` as well, in fact the `+1` is probably what gives me the extra block.